telegram/constants.py

```diff
@@ -62,6 +63,8 @@
     "PollType",
     "SUPPORTED_WEBHOOK_PORTS",
     "StickerType",
+    "TitleLimit",
+    "TopicLimit",
```

Comment:
I suggest to rename to `ForumTopic` since that's the term that's widely used in the API docs

Reply:
Yes, I thought of suggesting that too. You mean `ForumTopicLimit` instead of `TopicLimit`, right?

Reply:
yes, exactly :)

Reply:
Done